.github/workflows/build-push-image.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build Image
```

Comment:
Can this one be referenced instead of the one in right in `.github/workflows/publish-and-deploy.yaml`

Reply:
wow I completely forgot I wrote this